compiler_opt/rl/problem_configuration.py

```diff
@@ -113,5 +113,5 @@ def flags_to_add(self) -> Tuple[str, ...]:
   # List of flags to remove from clang compilation command. The flag names
   # should match the actual flags provided to clang.'
   def flags_to_delete(self) -> Tuple[str, ...]:
-    return ('-split-dwarf-file', '-split-dwarf-output', '-fthinlto-index',
-            '-fprofile-sample-use', '-fprofile-remapping-file')
+    return ('-split-dwarf-file', '-split-dwarf-output', '-fprofile-sample-use',
```

Comment:
Shouldn't `-fprofile-sample-use` and `-fprofile-remapping-file`  also be removed?

Reply:
i'm not sure, why shall we remove them?

Reply:
I think they should be there in delete_flags for default case (since they're not used?), but have to be removed from delete_flags and moved to replace_flags if we plan to use those flags for use-cases such as AFDO profile reinjection. I've done that in https://github.com/google/ml-compiler-opt/pull/103/files. PTAL.

Reply:
I left a comment on #103. We don't want to treat profile flags as delete by default. That's the whole point of modeling them as "replace": if present, and deleted, compilation works but the outcome is not what we want, and it's very hard to identify this as the root cause.

Debug info flags, in contrast, just don't matter to us right now in any scenario. Safe to treat as "delete" by default.